apps/req.c

```diff
@@ -951,7 +952,8 @@ static int prompt_info(X509_REQ *req,
 
     if (sk_CONF_VALUE_num(dn_sk)) {
         i = -1;
- start:for (;;) {
+ start:
+        for (;;) {
```

Comment:
since you edited the line, write it as ```for ( ; ; ) {``` -- that is, spaces around the semicolons.